README.md

```diff
@@ -155,9 +156,23 @@ writeCapabilities(
   path: 'fx51win10.json'
 )
 ```
+# ServiceBook Class
+## testProject
+Queries the Service Book project API for the given project `name`, iterates over
+its associated test repositories, checks them out from SCM, and executes their
+```run``` file(s).  Finally, it returns ```exit 0``` on successful/passing
+tests, and ```exit 1``` in the event of failed builds.
+
+## Examples
+```groovy
+def sb = new org.mozilla.fxtest.ServiceBook()  
```

Comment:
Sorry, I didn't mean to remove the library import, just the version tag as that will become dated.